frontend/src/store/RosController/index.js

```diff
@@ -90,18 +151,33 @@ class RosController {
   }
 
   @action
-  setIsWSConnected = (value) => {
+  setIsWSConnected = (value = false) => {
     this.isWSConnected = value
   }
 
   @action
-  setIsTeleopReady = (value) => {
+  setIsTeleopReady = (value = false) => {
     this.isTeleopReady = value
   }
 
   @action
-  setCurrentPosition = (position) => {
-    this.currentPosition = position
+  setBatteryState = (value = 100) => {
+    this.batteryState = (value | 0)
```

Comment:
as we have defined default `value` parametr we don't need to use `| 0`, because it won't be read
we should use one of approaches - either set default value or use `| 0`

Reply:
resolved